app/views/catalog/_show_letterbook.html.erb

```diff
@@ -23,11 +23,14 @@
         <article>
           <dl>
             <% document_show_fields(document).each do |field_name, field| -%>
-                <% if should_render_show_field? document, field %>
-                    <dt class="blacklight-<%= field_name.parameterize %>"><%= render_document_show_field_label document, field: field_name %></dt>
-                    <dd class="blacklight-<%= field_name.parameterize %>"><%= render_document_show_field_value document, field: field_name %></dd>
-                <% end -%>
+            <% if should_render_show_field? document, field %>
+            <dt class="blacklight-<%= field_name.parameterize %>"><%= render_document_show_field_label document, field: field_name %></dt>
+            <dd class="blacklight-<%= field_name.parameterize %>"><%= render_document_show_field_value document, field: field_name %></dd>
             <% end -%>
+            <% end -%>
+	    <dt>
+	      <%= link_to "Udgivelsen i Rex", "http://rex.kb.dk/KGL:KGL:KGL01" + @document['id'].to_s.match(/\/(\d+)\//)[1]  %>
```

Comment:
Maybe you should put this link in a div instead, to be separated from the metadata. Not a frontend fairy though :D